addons/point_of_sale/static/src/app/models/pos_order_line.js

```diff
@@ -731,6 +731,11 @@ export class PosOrderline extends Base {
     isSelected() {
         return this.order_id?.uiState?.selected_orderline_uuid === this.uuid;
     }
+    get canBeRemoved() {
+        const rounding = this.product_id.uom_id.rounding;
+        const decimals = Math.max(0, Math.ceil(-Math.log10(rounding)));
```

Comment:
```suggestion
        const decimals = this.models["decimal.precision"].find(
                    (dp) => dp.name === "Product Unit"
                ).digits;
```

get the number of decimals from the `Product Unit` decimal.precision record. Besides, `rounding` field of the `uom.uom` object is computed from it anyway.